src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/totrinnskontroll/AngreSendTilBeslutterService.kt

```diff
@@ -111,6 +111,10 @@ class AngreSendTilBeslutterService(
     private fun validerOppgave(
         saksbehandling: Saksbehandling,
     ) {
+        brukerfeilHvis(oppgaveService.hentBehandleSakOppgaveSomIkkeErFerdigstilt(saksbehandling.id) != null) {
```

Comment:
Hvis saksbehandler ofte får denne feilen kunne vi nesten vurdert om vi kunne stoppe/slette tasken for ferdigstillelse så de får angret med en gang. Men det får evt. være en forbedring til senere 🤓

Reply:
Ja 👍 
https://github.com/navikt/familie-prosessering-backend/pull/478 vil kunne gjøre det raskere.
Vi kan også vurdere andre løsninger, eks bedre håndtering for at "godkjenne-vedtak"-oppgaven ikke blir opprettet hvis man ønsker å angre. Det krevet litt mer kompleksitet